tests/test_packages/test_connections/test_local/test_search_services.py

```diff
@@ -399,29 +366,6 @@ def teardown_class(cls):
         cls.node.stop()
 
 
-class TestOEFConnectionFromJson:
```

Comment:
not supported anymore?

Reply:
No, because `Component.load_from_directory(ComponentType.CONNECTION, directory)` can be used.